db/migrate/20230907225057_create_theme_settings_migrations.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateThemeSettingsMigrations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :theme_settings_migrations do |t|
+      t.integer :theme_id, null: false
+      t.integer :theme_field_id, null: false
+      t.bigint :version, null: false
+      t.string :name, null: false, limit: 150
+      t.json :diff, null: false
```

Comment:
Good call on this column, I definitely think storing the diff will make our app more resiliant.